spark-rapids/test_spark_rapids.py

```diff
@@ -64,6 +64,9 @@ def test_spark_rapids(self, configuration, machine_suffixes, accelerator):
     if self.getImageVersion() == pkg_resources.parse_version("2.1") or self.getImageOs() == "rocky":
       self.skipTest("Not supported in image2.1 or rocky images")
 
+    if self.getImageVersion() == pkg_resources.parse_version("2.2") and self.getImageOs() == "debian":
+        self.skipTest("The Debian version (12) is not supported")
```

Comment:
The condition and the message don't match, in theory, the Debian version can change (because of fix for high-severity CVEs or something similar) and the message will become misleading. Either change the condition or the message. If the reason is about OS distro and version, it is better to check OS directly.

Also update other places.

Reply:
The message is exactly copied from the error logs from the test. But the tests are passing for debian 10 and debian 11. Any ideas on what to exactly do here?